waku/node/waku_node.nim

```diff
@@ -88,6 +90,8 @@ type
     wakuArchive*: WakuArchive
     wakuStore*: WakuStore
     wakuStoreClient*: WakuStoreClient
```

Comment:
See my comment further down on using `v3` in code. Perhaps rather rename these to `wakuLegacyStore`, `wakuLegacyStoreClient` ?